app/src/main/java/com/github/sdp/tarjetakuna/ui/browser/FilterState.kt

```diff
@@ -0,0 +1,18 @@
+package com.github.sdp.tarjetakuna.ui.browser
+
+/**
+ * Represents the state of the filter.
+ */
+data class FilterState(

```

Comment:
This data class is okay, but can be completely useless if we decide that the user can filter more than 1 thing. For example, I want to see only cards of cost X mana from set XXX. Maybe remove FilterType enum and add to the FilterState : filterMana, filterSet etc..